File: Objective-C/EntrypointSample-ObjC/ViewController.m
//
//  ViewController.m
//  EntrypointSample-ObjC
//
//  Created by 417.72KI on 2023/09/03.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    NSLog(@"ViewController loaded %@", [NSDate date]);
}

- (void)viewDidAppear:(BOOL)animated {
    [super viewDidAppear:animated];

    [NSThread sleepForTimeInterval:5];

    NSLog(@"Slept for 5 sec. %@", [NSDate date]);
}

@end

File: Swift-UIKit/Sources/EntrypointSample-Swift-UIKit/ViewController.swift
//
//  ViewController.swift
//  EntrypointSample-Swift-UIKit
//
//  Created by 417.72KI on 2023/09/03.
//

import UIKit

final class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()

        print("ViewController loaded", Date())
    }

    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)

        Thread.sleep(forTimeInterval: 5)
        print("Slept for 5 sec.", Date())
    }
}

